common/lib/xmodule/xmodule/course_module.py

```diff
@@ -919,7 +919,7 @@ class CourseModule(CourseFields, SequenceModule):  # pylint: disable=abstract-me
     """
 
 
-class CourseDescriptor(CourseFields, LicenseMixin, SequenceDescriptor):
+class CourseDescriptor(CourseFields, SequenceDescriptor, LicenseMixin):
```

Comment:
In particular, `XBlockMixins` have come at the end of the inheritance hierarchy (after `XBlock`)